application/classes/Ushahidi/Validator/Form/Update.php

```diff
@@ -31,12 +31,27 @@ public function __construct(FormRepository $repo)
 
 	protected function getRules()
 	{
+
+			// Always check validation for name and description
+			$fullData = $this->validation_engine->getFullData();
```

Comment:
With `not_empty` added to `Form/Update.php`, it was throwing the error when attempting to update the survey without changing the name. Adding this block was the first solution I found after looking at the validation workaround in `Post/Create.php`.